src/ZernikePolynomials.jl

```diff
@@ -97,8 +103,50 @@ struct OSA <: ZernikeIndex
     end
 end
 
-OSA(nm::NM) = ((m, n) = (nm.m, nm.n); OSA(Int((1//2)*(n*(n+2)+m))))
+function OSA(nm::NM)
+    m, n = nm.m, nm.n
+    j = (n * (n + 2) + m) ÷ 2
+    return OSA(j)
+end
+
+"""
+    Fringe(j::Integer)
+    Fringe(nm::NM)       # construct from NM index
+    Fringe(noll::Noll)   # construct from Noll index
+    Fringe(osa::OSA)     # construct from OSA index
+
+Set of 37 Zernike polynomials commonly used in optical design software and photolithography.
+
+Beware that the Fringe set is traditionally unnormalized.
```

Comment:
This seems a bit ambiguous; what does "unnormalized" even mean? And does this package return such polynomials, or not?

Conversely, I wonder if "Note that the normalization of the polynomials differs from the most common convention when using Fringe numbering schemes" might be a bit more explicit about what I think is your intent?